app/validators/questions/__init__.py

```diff
@@ -10,9 +10,13 @@
 from app.validators.questions.question_validator import QuestionValidator
 
 
-def get_question_validator(question):
+def get_question_validator(question, schema=None):
```

Comment:
Same as https://github.com/ONSdigital/eq-questionnaire-validator/pull/146#discussion_r913098639

Reply:
Removed and left only in `QuestionValidator` had to add some mocks in tests as consequence.